### PR TITLE
Updated Docs, Settings and quotes-js.py with ZYTE_SMARTPROXY.

### PR DESCRIPTION
**Pull request by mukthy**

### CODE REVIEW

File: splash_smart_proxy_manager_example/splash_smart_proxy_manager_example/settings.py [hunk 1]
```diff
@@ -12,9 +12,13 @@
     'scrapy_splash.SplashCookiesMiddleware': 723,
     'scrapy_splash.SplashMiddleware': 725,
     'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware': 810,
+    'scrapy_zyte_smartproxy.ZyteSmartProxyMiddleware': 610
 }
 
-CRAWLERA_APIKEY = ''  # Your crawlera API key
+ZYTE_SMARTPROXY_ENABLED = True
```
Comment (ThriveniPatil) [inline splash_smart_proxy_manager_example/splash_smart_proxy_manager_example/settings.py]: Zyte_Smartproxy_enabled is this required? previous it was not required. we can just have the SPM APIkey.

**Comment by mukthy** [comment]
Adding an extra comments with Results.
image

File: splash_smart_proxy_manager_example/splash_smart_proxy_manager_example/settings.py (hunk 1 above)
Comment (mukthy) [reply splash_smart_proxy_manager_example/splash_smart_proxy_manager_example/settings.py]: @ThriveniPatil if we do not set the Zyte_Smartproxy_enabled to true then the request from splash is not going through the Smart Proxy Manager. Attached are my test results with the ZYTE_SMARTPROXY_ENABLED commented out.

image

**Comment by mukthy** [comment]
As discussed, I have made the changes. 

Zyte_SmartProxy_Enabled=True (removed)
ZyteSmartProxyManager middleware (removed)

**Pull request merged**